### PR TITLE
[FREELDR] DetectPnpBios(): Add 'i' to an ERR()

### PR DESCRIPTION
**Pull request by SergeGautherie**
Addendum to 17990b2 (r73617).
JIRA issue: [CORE-12623](https://jira.reactos.org/browse/CORE-12623)

### CODE REVIEW

File: boot/freeldr/freeldr/arch/i386/pc/machpc.c [hunk 1]
```diff
@@ -273,7 +273,7 @@ DetectPnpBios(PCONFIGURATION_COMPONENT_DATA SystemKey, ULONG *BusNumber)
 
             if (PnpBufferSize + DeviceNode->Size > PnpBufferSizeLimit)
             {
-                ERR("Buffer too small! Ignoring remaining device nodes.\n");
+                ERR("Buffer too small! Ignoring remaining device nodes. (i = %d)\n", i);
```
Comment (binarymaster) [inline boot/freeldr/freeldr/arch/i386/pc/machpc.c:276]:
```suggestion
                ERR("Buffer too small! Ignoring remaining device nodes. (NodeNumber: %d)\n", i);
```

File: boot/freeldr/freeldr/arch/i386/pc98/pc98hw.c [hunk 2]
```diff
@@ -1150,7 +1150,7 @@ DetectPnpBios(PCONFIGURATION_COMPONENT_DATA SystemKey, ULONG *BusNumber)
 
             if (PnpBufferSize + DeviceNode->Size > PnpBufferSizeLimit)
             {
-                ERR("Buffer too small! Ignoring remaining device nodes.\n");
+                ERR("Buffer too small! Ignoring remaining device nodes. (i = %d)\n", i);
```
Comment (binarymaster) [inline boot/freeldr/freeldr/arch/i386/pc98/pc98hw.c:1153]:
```suggestion
                ERR("Buffer too small! Ignoring remaining device nodes. (NodeNumber: %d)\n", i);
```

File: boot/freeldr/freeldr/arch/i386/pc/machpc.c (hunk 1 above)
Comment (SergeGautherie) [reply boot/freeldr/freeldr/arch/i386/pc/machpc.c:276]: It looks like `i` and `NodeNumber` values may not be identical, so should not be mixed.
Do you want both?

File: boot/freeldr/freeldr/arch/i386/pc98/pc98hw.c (hunk 2 above)
Comment (SergeGautherie) [reply boot/freeldr/freeldr/arch/i386/pc98/pc98hw.c:1153]: (Same case.)

File: boot/freeldr/freeldr/arch/i386/pc/machpc.c (hunk 1 above)
Comment (binarymaster) [reply boot/freeldr/freeldr/arch/i386/pc/machpc.c:276]: Ok I see, let's leave `i` only then.

Comment (HBelusca) [reply boot/freeldr/freeldr/arch/i386/pc/machpc.c:276]: Can you explain to me why i and NodeNumber may not be identical?

Comment (binarymaster) [reply boot/freeldr/freeldr/arch/i386/pc/machpc.c:276]: Passed by pointer in `PnpBiosGetDeviceNode` call, though I didn't looked into it.

**Review by HBelusca — APPROVED** [review]

**Pull request merged**